.github/workflows/build_rpm.yaml

```diff
@@ -0,0 +1,174 @@
+---
+name: Build RPM pipeline
+on:
+  workflow_call:
+    inputs:
+      target-architecture:
+        description: 'What is the architecture we want to build for'
+        required: true
+        type: string
+      build-version:
+        description: 'What is the version used to build and tag the RPMs'
+        required: true
+        type: string
+      release-version:
+        description: 'What is the release version used to build and tag the RPMs'
+        required: true
+        type: string
+      tag:
+        description: 'The tag used to checkout the correct version of source code'
+        required: false
+        type: string
+        default: ''
+
+
+jobs:
+  parsing:
+    runs-on: ubuntu-latest
+    outputs:
+      name: ${{ steps.params.outputs.name }}
+    steps:
+
+      - uses: actions/checkout@v4
+        with:
+          repository: alumet-dev/packaging
+          fetch-tags: 'true'
+          ref: 'ci/create-rpm-package'  # TODO Change on merge
+          path: 'packaging'
+
+      - name: Parse alumet version
```

Comment:
I don't get the point of this step, why not use hardcoded `alumet` as name in other steps ? Why would it varies ?

Also if we keep it this way I think you could change the name of step to `Parse alumet package name`